nuitka/build/include/nuitka/exceptions.h

```diff
@@ -441,8 +441,16 @@ NUITKA_MAY_BE_UNUSED static inline void ADD_EXCEPTION_CONTEXT(PyObject **excepti
 }
 #endif
 
-// Special helper that checks for StopIteration and if so clears it, only
-// indicating if it was set.
+/* Special helper that checks for StopIteration and if so clears it, only
```

Comment:
Very well. Just for my information, did you check how these things look in Doxygen? To my shame I didn't look at the C part at all yet. I was wondering where Doxygen expects comments to pick them up. So far Nuitka has followed no rules.

Reply:
I will look into it.

Reply:
It works fine. :smile: